backends/qualcomm/qnn_quantizer.py

```diff
@@ -153,21 +104,17 @@ def get_default_qnn_ptq_config(
         bias=bias_quantization_spec,
     )
 
-    quantizer_configs = QnnQuantizerConfig(
-        enable_per_channel_conv_quant=enable_per_channel_conv_quant
-    )
-
-    return quantization_config, quantizer_configs
+    return quantization_config
 
 
-def get_16bit_qnn_ptq_config() -> Tuple[QuantizationConfig, QnnQuantizerConfig]:
+def get_default_16bit_qnn_ptq_config() -> QuantizationConfig:
```

Comment:
just curious, does qnn library mentioned in the recipe (https://pytorch.org/executorch/stable/build-run-qualcomm-ai-engine-direct-backend.html) supports 4 bit quantized linear?

Reply:
4 bit quantization is still WIP. We will introduce them to here once it finished.

Reply:
do we know an estimated timeline for this?

Reply:
Sorry that it might take a while to estimate the time. Because we are in the very early stage of investigation. we're still trying to understand things like QNN INT4 support coverage currently.